src/Orleans/Messaging/Message.cs

```diff
@@ -433,7 +433,7 @@ private Message(Categories type, Directions subtype)
         // Caller must clean up bytes
         public Message(List<ArraySegment<byte>> header)
         {
-            var context = new DeserializationContext
+            var context = new DeserializationContext(RuntimeClient.Current.InternalGrainFactory)
```

Comment:
Clearly this isn't the desired end-state, but these ctors are changed in the subsequent SerializationManager PR